### DISCUSSION TITLE
教授好難搞

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#2237] [General]
最近加入了實驗室，一開始聽說我們教授很兇對教授的印象沒有很好，但在去問過後並沒有傳言中那麼恐怖，實際上進去後也確實沒看過他罵人，實驗室的氛圍我也覺得不錯，不過我們教授會要求我們要對實驗室有貢獻，但我每週一到五都有去實驗室，而且在學過材料製程後也都有在幫忙長材料，老師要求而學長姐不想做的事我們也有幫忙弄，我認為我已經算是有貢獻了，然而他還是覺得我沒在做事，認為我對不起實驗室的六學分。明明我已經很努力的在幫忙，而且也沒有弄壞什麼儀器，他為什麼要這樣？

**Comment by ed112601513** [comment]
我也是這個學期加了個實驗室，性質比較像是純粹幫忙而已，所以教授沒有要求我做太多事情，實驗室的氛圍很好，我只要盡力做好該做的事，遇到問題就去找助理討論，到現在壓力不算大，也可能是因為我沒有拿學分才可以貢獻度這麼低還沒有被嫌棄吧。

**Comment by Chattenoire** [comment]
老教授們就跟老體育老師們一樣，劣多良少。
只能說盡量去找年輕教授吧，那些快退休的自恃沒人動得了，常常沒有下限的為所欲為，根本不懂"教授"二字的涵義，只會把這個職位當作行使權力的底氣，超級可悲。

**Comment by Taoyi-Chiang** [comment]
之前當助理遇到一個連最基本的量化方法，敘述統計都搞不清楚的人，一直強調要我做一張圖只有一根柱子的直方圖，做了跟我說這不是她要的，可是我當天確認了三次。
然後整天一直抱怨現在的學生怎樣糟糕，問我意見，我都搞不清楚我是去工作還是去諮商治療，亂七八糟什麼東西。
說到問我意見，他也很愛問我意見，要怎麼樣讓學生喜歡上課，我提了一些意見想法，又開始說這不好那不好，可是又會怎樣怎樣。
說到他的研究報告，他一直說我不能只做我自己的事講了一個月，但結案最後一週才給我一堆raw data，然後此時還在要我做一堆廢是不事。
這種人真的有夠可悲，30幾歲畢業前就在石化業大學教，教到現在快退休還在教，出道即巔峰。
這種垃圾真的快點去死，不橫死街頭都對不起養他的爸媽欸，趕快把自己收一收好嗎？不要出來禍害人間了好不好。
